Python/bytecodes.c

```diff
@@ -2019,14 +2015,10 @@ dummy_func(
         }
 
         inst(POP_JUMP_IF_FALSE, (cond -- )) {
-            if (Py_IsTrue(cond)) {
-                _Py_DECREF_NO_DEALLOC(cond);
-            }
-            else if (Py_IsFalse(cond)) {
-                _Py_DECREF_NO_DEALLOC(cond);
+            if (Py_IsFalse(cond)) {
```

Comment:
Will this cause a performance change, since formerly `IsTrue` then `IsFalse`, now `IsFalse` then `IsTrue`

Though this comparison is lightweight.

Reply:
Honestly, I doubt it.